src/spxmod/regmod_builder.py

```diff
@@ -38,25 +48,109 @@ def msca_optimize(
     return result.x
 
 
+class SparseParameter(Parameter):
```

Comment:
Could there be any reason to add `SparseParameter` directly to regmod?

Reply:
Yes! These modules are temporary in nature, until we find good place for them :)